recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,176 @@
+import os
+import re
+import shutil
+
+from fnmatch import fnmatch
+from conans import ConanFile, CMake, tools
+
+class VTKConan(ConanFile):
+    name = "vtk"
+    version = "9.0.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://vtk.org/"
+    license = "BSD license"
+    description = "Visualization Toolkit by Kitware"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    source_subfolder = "source_subfolder"
+    options = {"shared": [True, False], "qt": [True, False], "mpi": [True, False],
```

Comment:
Most recipes follow the following convention:
- when an option conditionally adds an external requirement, we use `with_XXX`. In this case, `qt` should become `with_qt`.
- when an option conditionally enabled a feature, you can just use `XX`. e.g.: `"feature": [True, False]`.

Reply:
`qt` replaced by `with_qt`, but VTK expert needed here to verify name correctness for other switches

Reply:
Question duplicated in other place.
Resolving here.